schema/bom-1.6.schema.json

```diff
@@ -938,7 +938,7 @@
             "excluded"
           ],
           "meta:enum": {
-            "required": "The component is required for runtime",
+            "required": "The component is required for runtime, or another version.",
```

Comment:
this would make a MUST a MAY.

Spec-wise, i dislike the change. it is a breaking change from OBOM's point of viev:
In an OBOM a required component has a specific version.
with this change, spec-wise, this would be no longer the case.